### PR TITLE
docs: update to reflect graphql API changes

### PR DESCRIPTION
**Pull request by xbcsmith**

### CODE REVIEW

**Review by caproven — DISMISSED** [review]

File: docs/tutorials/hello_world/README.md [hunk 1]
```diff
@@ -175,12 +175,18 @@ the previously created event
...
     name
     version
+    release
+    platform_id
+    package
     description
     payload
     success
+    event_receiver_id
+		created_at
```
Comment (caproven) [inline docs/tutorials/hello_world/README.md:189]:
```suggestion
    created_at
```

File: docs/how-to/start-server/README.md [hunk 2]
```diff
@@ -148,10 +158,18 @@ the previously created event
...
-  event(id: "01HEK4REW0S93V8Y10E8A8M8HC") {
+  events(id: "01HKNDTSFT6ZZ8Q8YNK736TT43") {
+    id
     name
     version
-    description
+    release
+    platform_id
+    package
+		description
```
Comment (caproven) [inline docs/how-to/start-server/README.md:168]:
```suggestion
    description
```

**Review by caproven — COMMENTED** [review]
missed a couple indentation spots in my last review

File: docs/how-to/start-server/README.md [hunk 3]
```diff
@@ -148,10 +158,18 @@ the previously created event
...
     name
     version
+    release
+    platform_id
+    package
     description
+    payload
+    success
+    event_receiver_id
+		created_at
```
Comment (caproven) [inline docs/how-to/start-server/README.md:172]:
```suggestion
    created_at
```

**Review by ninjamast3r — DISMISSED** [review]

**Review by ninjamast3r — APPROVED** [review]

**Review by gbuskey — APPROVED** [review]

**Review by jewelsf — APPROVED** [review]

**Pull request merged**